charger/openevse.go

```diff
@@ -197,7 +197,8 @@ func (c *OpenEVSE) Enabled() (bool, error) {
 		}
 	}
 
-	return false, errors.New("invalid EVSE state")
+	// no override:
+	return false, c.Enable(false)
```

Comment:
Könntest Du das erläutern? In der Abfrage des Status sollte eigentlich nicht geschaltet werden!

Reply:
The problem: OpenEVSE has multiple options and can be on standby due to scheduled charging, solar divert, etc.
Solutions:
- report the current state, even if that might change
- enforce a consistent state through a manual override (the same would happen once you'd start charging)

I'd like the implementation to follow the best practices, so please advise.

Reply:
I went with the first option and it works correctly.
If the EVSE goes out of sync, it's brought back in sync through an override.

Reply:
> I went with the first option and it works correctly.

That's perfectly in line with the rest of evcc logic